tests/ext/pcntl/pcntl_fork_long_running_manual.phpt

```diff
@@ -0,0 +1,44 @@
+--TEST--
+Long running manual flush
+--ENV--
+DD_TRACE_CLI_ENABLED=true
+DD_TRACE_GENERATE_ROOT_SPAN=false
+DD_TRACE_AUTO_FLUSH_ENABLED=false
+--FILE--
+<?php
+
+require 'functions.inc';
+require getenv('REQUEST_INIT_HOOK_PATH');
```

Comment:
Is there anything that this test is covering that isn't covered in the PhpUnit tests? If not, is there a way we can move this test to the PhpUnit tests? The `phpt` tests are separated from the userland code on purpose and I'm not sure we want to use the real request init hook to start brining in the userland dependency, especially since we are in the process of moving way from userland altogether.

Reply:
As a consequence of using the fake classes, I am no longer depending anymore on the real init hook. Thanks for the TIL :). Here is how these tests changed https://github.com/DataDog/dd-trace-php/pull/1196/commits/c0fc15a917c3a0342f1a8d89f8ea378c77e883ca